src/components/molecules/Settings/Account/AccountSection/index.tsx

```diff
@@ -9,32 +9,50 @@ import { useIntl } from "react-intl";
 
 export type Props = {
   email?: string;
+  appTheme?: string;
   lang?: string;
   hasPassword: boolean;
   updatePassword?: (password: string, passwordConfirmation: string) => void;
   updateLanguage?: (lang: string) => void;
+  updateTheme?: (lang: string) => void;
 };
 
 const items = [
   { key: "ja", label: "日本語" },
   { key: "en", label: "English" },
 ];
 
+const themeItems = [
+  { key: "DARK", label: "Dark theme", icon: "moon" },
+  { key: "LIGHT", label: "Light theme", icon: "sun" },
+];
+
 const ProfileSection: React.FC<Props> = ({
   email,
+  appTheme,
   hasPassword,
   updatePassword,
   updateLanguage,
+  updateTheme,
 }) => {
   const intl = useIntl();
   const [isOpen, setIsOpen] = useState(false);
   const [currentLang, setCurrentLang] = useState<string>();
+  const [currentThemeValue, setCurrentThemeValue] = useState<string>();
+  const [currentThemeLabel, setCurrentThemeLabel] = useState<string>();
 
   useEffect(() => {
     const lang = items.find(item => item.key === intl.locale);
     setCurrentLang(lang?.label);
   }, [intl.locale]);
 
+  useEffect(() => {
+    const appThemeValue = appTheme ? appTheme : "DARK";
```

Comment:
Just a small change, but you could just move this logic into the useState declaration like:
```
const [currentThemeValue, setCurrentThemeValue] = useState(appTheme ?? "DARK");
```
Then, all this useEffect is doing is the updating of the value. The label is a little different so leaving it as is seems appropriate.